internal/test-utils/src/jvmCommonMain/kotlin/coil3/test/utils/AbstractSvgDecoderTest.kt

```diff
@@ -1,91 +1,69 @@
-package coil3.decode
+package coil3.test.utils
 
-import coil3.BitmapImage
+import coil3.Image
 import coil3.ImageLoader
+import coil3.decode.DataSource
+import coil3.decode.Decoder
+import coil3.decode.ImageSource
 import coil3.fetch.SourceFetchResult
 import coil3.request.Options
 import coil3.size.Dimension
 import coil3.size.Scale
 import coil3.size.Size
-import coil3.test.utils.assertIsSimilarTo
-import coil3.test.utils.context
-import coil3.test.utils.decodeBitmapAsset
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertIs
 import kotlin.test.assertNotNull
 import kotlin.test.assertNull
 import kotlin.test.assertTrue
 import kotlinx.coroutines.test.runTest
-import okio.Buffer
 import okio.BufferedSource
 import okio.FileSystem
+import okio.Path.Companion.toPath
 import okio.buffer
-import okio.source
-import org.junit.Test
 
-class SvgDecoderTest {
+abstract class AbstractSvgDecoderTest(
```

Comment:
Can this class live in `coil-svg/jvmCommonTest` instead of moving it to `test-utils`?

Reply:
When it was in `jvmCommonTest` it was not visible to android instrumented test.